src/main/resources/reference.conf

```diff
@@ -76,6 +79,18 @@ cassandra-journal {
   # The time to wait before cassandra will remove the thombstones created for deleted entries.
   # cfr. gc_grace_seconds table property documentation on http://www.datastax.com/documentation/cql/3.1/cql/cql_reference/tabProp.html
   gc-grace-seconds = 864000
+  
+  # Configure know tags in this section
```

Comment:
My idea was to cluster on tag and duplicate the event with multiple tags. That of course would mean we would have to reconcile the event on reads and replay which is not ideal, but should not be a major issue. 

If we could partition on a collection life would be so much easier. But the above solution would internally be essentially the same, it would just require some work on our side.

What I suggest we do is stick with 3 for now, I think it is reasonable tradeoff. I have started an attempt to do the above a few days ago so I could try if that works and potentially create a separate PR.